doc/user-guide.adoc

```diff
@@ -514,6 +514,87 @@ To create a secured HTTPS route, see the link:++#certificate-manager-integration
 
 _This feature is only available if you are running on OKD or OpenShift._
 
+
+==== Non-Knative deployment (Ingress)
+
+Before you can use the Ingress resource to expose your cluster, you must install an ingress controller, such a Nginx or Traeik..
+
+_The Ingress resource is created only if the `Route` resource is not available._
+
+
+To use the Ingress resource, set the `defaultHostName` variable in the runtime-component-operator ConfigMap object to a host name such as mycomp.com
```

Comment:
It'll be good to italicize values such as _runtime-component-operator_ and _mycomp.com_